test/__snapshots__/loader.test.js.snap

```diff
@@ -349,6 +349,30 @@ Object {
 
 exports[`loader should work from esModule export: warnings 1`] = `Array []`;
 
+exports[`loader should work inline 1 with config loader options: errors 1`] = `Array []`;
+
+exports[`loader should work inline 1 with config loader options: module 1`] = `
+"var ___EXPOSE_LOADER_IMPORT___ = require(\\"-!../../node_modules/babel-loader/lib/index.js??ref--4-0!./custom.js?foo=bar\\");
```

Comment:
Copied one of the inline tests and added options to the babel loader config. The result is this `"??ref--4-0"` reference added to the babel loader part of the expose loader import.